content/reproducibility.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "The case for reproducibility"
+author: "Lindsay R. Carr"
+slug: "Reproducibility"
+date: "2016-10-04"
+description: "Explanation of reproducibility and resources to start practicing it"
+keyword1: "reproducibility"
+keyword2: "reproducible science"
+keyword3: "spreadsheets"
+output: USGSmarkdowntemplates::hugo
+---
+
+## Science is hard, why make it harder?
+
+Scientists and researchers spend a lot of time on data preparation and analysis, and some of these analyses are quite computationally intensive. The amount of time required to conduct an analysis grows for the increased complexity of calculations, amount of data, and number of datasets that will be analyzed. Many researchers use spreadsheets to conduct their analysis workflows, and apply the same set to each data set manually. A majority of the time in this type of workflow could be spent copying and pasting equations from one spreadsheet or column to another. This spreadsheet workflow is highly error-prone, as *The Economist* shows in "[Excel errors and science papers](http://www.economist.com/blogs/graphicdetail/2016/09/daily-chart-3?fsrc=scn/fb/te/bl/ed/excelerrorsandsciencepapers)".
```

Comment:
I agree, I think adding this is a little off topic since it's discussing genomics, and isn't advocating for scripting as an alternative.